Classes/Command/SupervisorCommandController.php

```diff
@@ -56,7 +46,65 @@ public function createCommand(): void
                 );
             }
         }
+    }
+
+    public function startGroupsCommand(): void
+    {
+        $groups = $this->getGroups();
+        array_walk($groups, fn(Model\Group $group) => $this->runSupervisorCommand($group, 'start'));
```

Comment:
arrow functions require a minimum PHP version of 7.4. Since all versions prior to 7.4 are eol by now, I'd be fine with dropping support for older PHP versions in this package.

This needs to be declared in the composer.json and the GitHub workflows have to be adjusted accordingly.

Reply:
Somehow the usage of the arrow functions also makes the unit tests fail with PHP 7.4 and Flow 5.3 in the stage of generating the coverage report. I could reproduce that in tests on my local machine.
```
$ bin/phpunit -c DistributionPackages/Netlogix.Supervisor/phpunit.xml.dist --testsuite="Unit" --bootstrap "Build/BuildEssentials/PhpUnit/UnitTestBootstrap.php"
PHPUnit 6.0.13 by Sebastian Bergmann and contributors.

.......                                                             7 / 7 (100%)

Time: 33 ms, Memory: 4.00MB

OK (7 tests, 8 assertions)

Generating code coverage report in Clover XML format ...%

$ echo $?
255
```

When reverting to the old code before using arrow functions everything runs well.
Should we give up using arrow functions? Otherwise we have do debug what happens during coverage report generation and if it is an issue with phpunit or xdebug.

Reply:
I'm fine with switching to the non-arrow function closure syntax. 👍